<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glcapsviewerClass</class>
 <widget class="QMainWindow" name="glcapsviewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenGL Hardware Capability Viewer 1.1 - Â© 2011-2015 by Sascha Willems (www.saschawillems.de)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="glcapsviewer.qrc">
    <normaloff>:/glcapsviewer/Resources/glCapsViewer.ico</normaloff>:/glcapsviewer/Resources/glCapsViewer.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolBar {
	background-image: url(:/glcapsviewer/Resources/OpenGL48.png);
	background-repeat:no;
	background-position:right;
	border:0px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabDevice">
       <attribute name="icon">
        <iconset resource="glcapsviewer.qrc">
         <normaloff>:/glcapsviewer/Resources/gpu32.png</normaloff>:/glcapsviewer/Resources/gpu32.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Your device</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelDescription">
            <property name="styleSheet">
             <string notr="true">font: 75 11pt;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>Getting OpenGL implementation details...</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelReportPresent">
            <property name="styleSheet">
             <string notr="true">font: 10pt;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QTreeWidget" name="treeWidget">
              <property name="styleSheet">
               <string notr="true">QTreeView::item { height: 24px;}</string>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="indentation">
               <number>10</number>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>200</number>
              </attribute>
              <attribute name="headerStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>key</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>value</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTabWidget" name="tabWidgetDevice">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>Extensions</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QTreeWidget" name="treeWidgetExtensions">
                  <property name="styleSheet">
                   <string notr="true">QTreeView::item { height: 24px;}</string>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::NoEditTriggers</set>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <attribute name="headerVisible">
                   <bool>false</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string notr="true">1</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>Compressed texture formats</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QListWidget" name="listWidgetCompressedFormats">
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_3">
               <attribute name="title">
                <string>Internal formats</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QTreeWidget" name="treeWidgetInternalFormats">
                  <property name="styleSheet">
                   <string notr="true">QTreeView::item { height: 24px;}</string>
                  </property>
                  <property name="editTriggers">
                   <set>QAbstractItemView::NoEditTriggers</set>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="indentation">
                   <number>10</number>
                  </property>
                  <attribute name="headerDefaultSectionSize">
                   <number>250</number>
                  </attribute>
                  <attribute name="headerStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>key</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>value</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDatabase">
       <attribute name="icon">
        <iconset resource="glcapsviewer.qrc">
         <normaloff>:/glcapsviewer/Resources/db24.png</normaloff>:/glcapsviewer/Resources/db24.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Database</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item alignment="Qt::AlignLeft">
           <widget class="QPushButton" name="pushButtonRefreshDataBase">
            <property name="text">
             <string>Refresh</string>
            </property>
            <property name="icon">
             <iconset resource="glcapsviewer.qrc">
              <normaloff>:/glcapsviewer/Resources/refresh16.png</normaloff>:/glcapsviewer/Resources/refresh16.png</iconset>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QListWidget" name="listWidgetDatabaseDevices">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>Please refresh...</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true">font: 75;
color: rgb(0, 0, 255);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="text">
                 <string>Report version</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxDeviceVersions"/>
              </item>
              <item>
               <widget class="QLabel" name="labelDatabaseDeviceImplementation">
                <property name="styleSheet">
                 <string notr="true">font: 75;
color: rgb(0, 0, 255);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="text">
                 <string>Implementation</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tableWidgetDatabaseDeviceReport">
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="showGrid">
                 <bool>false</bool>
                </property>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>Capability</string>
                 </property>
                 <property name="font">
                  <font>
                   <weight>50</weight>
                   <bold>false</bold>
                   <underline>false</underline>
                   <kerning>true</kerning>
                  </font>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Value</string>
                 </property>
                 <property name="font">
                  <font>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelDatabaseDeviceExtensions">
                <property name="styleSheet">
                 <string notr="true">font: 75;
color: rgb(0, 0, 255);</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="text">
                 <string>Extensions</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="listWidgetDatabaseDeviceExtensions">
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionUpload"/>
   <addaction name="actionSave_xml"/>
   <addaction name="separator"/>
   <addaction name="actionDevice"/>
   <addaction name="actionDatabase"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="glcapsviewer.qrc">
     <normaloff>:/glcapsviewer/Resources/refresh48.png</normaloff>:/glcapsviewer/Resources/refresh48.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refrehs OpenGL report</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset resource="glcapsviewer.qrc">
     <normaloff>:/glcapsviewer/Resources/upload48.png</normaloff>:/glcapsviewer/Resources/upload48.png</iconset>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
   <property name="toolTip">
    <string>Upload OpenGL report to online database</string>
   </property>
  </action>
  <action name="actionDatabase">
   <property name="icon">
    <iconset resource="glcapsviewer.qrc">
     <normaloff>:/glcapsviewer/Resources/browse48.png</normaloff>:/glcapsviewer/Resources/browse48.png</iconset>
   </property>
   <property name="text">
    <string>Database</string>
   </property>
   <property name="toolTip">
    <string>Display OpenGL online database in default browser</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="glcapsviewer.qrc">
     <normaloff>:/glcapsviewer/Resources/settings48.png</normaloff>:/glcapsviewer/Resources/settings48.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Application settings (Proxy)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="glcapsviewer.qrc">
     <normaloff>:/glcapsviewer/Resources/about48.png</normaloff>:/glcapsviewer/Resources/about48.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Application information</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="glcapsviewer.qrc">
     <normaloff>:/glcapsviewer/Resources/close48.png</normaloff>:/glcapsviewer/Resources/close48.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit OpenGL caps viewer</string>
   </property>
  </action>
  <action name="actionSave_xml">
   <property name="icon">
    <iconset resource="glcapsviewer.qrc">
     <normaloff>:/glcapsviewer/Resources/save48.png</normaloff>:/glcapsviewer/Resources/save48.png</iconset>
   </property>
   <property name="text">
    <string>Save xml</string>
   </property>
   <property name="toolTip">
    <string>Export current OpenGL report to xml</string>
   </property>
  </action>
  <action name="actionDevice">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="glcapsviewer.qrc">
     <normaloff>:/glcapsviewer/Resources/device48.png</normaloff>:/glcapsviewer/Resources/device48.png</iconset>
   </property>
   <property name="text">
    <string>Device</string>
   </property>
   <property name="toolTip">
    <string>Show device in database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="glcapsviewer.qrc"/>
 </resources>
 <connections/>
</ui>
